### PR TITLE
SDI-4: ✨ Add nomis prisoner environments and tidy up

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: dashboards/circle.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <div data-id="circle-ci-ministryofjustice-hmpps-auth-main" data-view="CircleCi" data-title="Auth Server"></div>
     </li>
     <li data-row="1" data-col="3" data-sizex="1" data-sizey="1">
-      <div data-id="circle-ci-ministryofjustice-keyworker-service-main" data-view="CircleCi" data-title="Keyworker Api"></div>
+      <div data-id="circle-ci-ministryofjustice-keyworker-api-main" data-view="CircleCi" data-title="Keyworker API"></div>
```
Comment (petergphillips) [inline dashboards/circle.erb:18]: renamed service

File: dashboards/circle.erb [hunk 2]
```diff
@@ -68,9 +68,6 @@
       <div data-id="circle-ci-ministryofjustice-hmpps-restricted-patients-api-main" data-view="CircleCi" data-title="HMPPS Restricted Patients API"></div>
     </li>
 
-    <li data-row="4" data-col="1" data-sizex="1" data-sizey="1">
-      <div data-id="circle-ci-ministryofjustice-offender-assessments-api-master" data-view="CircleCi" data-title="Offender Assessment API"></div>
```
Comment (petergphillips) [inline dashboards/circle.erb:72]: project archived

File: jobs/health.rb [hunk 3]
```diff
@@ -75,23 +75,23 @@
     {name: 'hmpps-restricted-patients-api', versionUrl: 'https://restricted-patients-api.hmpps.service.justice.gov.uk/info', url: 'https://restricted-patients-api.hmpps.service.justice.gov.uk/health'},
     {name: 'hmpps-welcome-people-into-prison-api', versionUrl: 'https://welcome-api.prison.service.justice.gov.uk/info', url: 'https://welcome-api.prison.service.justice.gov.uk/health' },
     {name: 'hmpps-welcome-people-into-prison-ui', url: 'https://welcome.prison.service.justice.gov.uk/health'},
-    {name: 'prison-to-nhs-update', versionUrl: 'https://prison-to-nhs-update-dev.prison.service.justice.gov.uk/info', url: 'https://prison-to-nhs-update-dev.prison.service.justice.gov.uk/health'},
+    {name: 'prison-to-nhs-update', versionUrl: 'https://prison-to-nhs-update.prison.service.justice.gov.uk/info', url: 'https://prison-to-nhs-update.prison.service.justice.gov.uk/health'},
```
Comment (petergphillips) [inline jobs/health.rb:78]: typo here since it is production services

**Review by Mjwillis — APPROVED** [review]
Lgtm

**Pull request merged**